sdk/appconfiguration/app-configuration/test/internal/tracingHelpers.spec.ts

```diff
@@ -172,15 +175,15 @@ describe("tracingHelpers", () => {
 
     assert.deepEqual(traceData, {
       operationName: "listConfigurationSettings",
-      options: { ...operationOptions, keyFilter: "ignored" }
+      options: { ...operationOptions, keyFilter: "ignored" } as OperationOptions
```

Comment:
why `as ...`?

Reply:
Because `{ ...operationOptions, keyFilter: "ignored" }` wasn't being recognized as an `OperationOptions` object, and so there was an error. Do you know a way on how to avoid explicitly casting as `OperationOptions` in here?

Reply:
Shouldn't you be updating the traceData type since it now has a new property?

Reply:
![image](https://user-images.githubusercontent.com/10452642/145914047-943f450c-0da3-4f91-aeb0-acfb89915d82.png)
Instead of just OperationOptions, make it OperationOptions & {keyFilter ?: string} ?

Reply:
Done :) Changed to `OperationOptions & {keyFilter ?: string}`